HotSpot/View Controllers/LoginViewController.h

```diff
@@ -0,0 +1,20 @@
+//
+//  LoginViewController.h
+//  HotSpot
+//
+//  Created by aaronm17 on 7/16/19.
+//  Copyright © 2019 aodemuyi. All rights reserved.
+//
+
+#import <UIKit/UIKit.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface LoginViewController : UIViewController
+
+@property (weak, nonatomic) IBOutlet UITextField *username;
```

Comment:
why are these IBOutlets in the header file? I think these can be in the interface of the implementation file.